.actions/setup_tools.py

```diff
@@ -269,7 +271,8 @@ def _load_aggregate_requirements(req_dir: str = "requirements", freeze_requireme
     if not requires:
         return None
     # TODO: add some smarter version aggregation per each package
-    requires = list(chain(*requires))
+    requires = set(chain(*requires))
+    requires = [req for req in requires if list(parse_requirements([req]))[0].name not in skip_pkgs]
```

Comment:
This would work too?

```suggestion
    requires = [req for req in requires if req.replace("-", "_") not in _PACKAGE_MAPPING.values()]
```

Reply:
Note that `req` here also include versions